src/main/java/org/folio/oaipmh/helpers/AbstractGetRecordsHelper.java

```diff
@@ -534,7 +534,7 @@ protected Future<JsonObject> requestFromInventory(Request request, int limit, St
     var querySuppressFromDiscovery = discoverySuppress ? " and discoverySuppress==" + suppressedRecordsSupport :
       EMPTY;
 
-    String query = "limit=" + limit + "&query=" +
+    String query = "limit=" + limit + "&offset=" + request.getOffset() + "&query=" +
```

Comment:
Why do we have code changes in this PR? Is it relating to some stories from NEWS.md?

Reply:
Yes, it is relating to MODOAIPMH-138